config/main.py

```diff
@@ -315,7 +316,11 @@ def _restart_services():
     ]
     for service in services:
         try:
-            run_command("systemctl restart %s" % service, display_cmd=True)
+            # We first run "systemctl reset-failed" to ensure that we
+            # can restart the service, even if it has entered a failed state
+            click.echo("Restarting {} ...".format(service))
+            run_command("systemctl reset-failed {}".format(service))
```

Comment:
I assume the error code returned by reset-failed if the services is not in failed state, doesn't trigger the exception. Looks good to me otherwise.

Reply:
Error code shouldn't trigger the exception. Plus, calling `systemctl reset-failed` on a service not in a failed state will still return 0.

Reply:
Great. Thank you for fixing this.